examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -214,9 +187,12 @@ void main([List<String> args = const <String>[]]) {
       final String histogramPath = path.join(testOutputsDirectory, 'transition_durations.timeline.json');
       await saveDurationsHistogram(timeline.json['traceEvents'], histogramPath);
 
+      // Scroll back to the top
+      await driver.scroll(find.byType('CustomScrollView'), 0.0, 1000.0, const Duration(milliseconds: 100));
```

Comment:
Maybe do `scrollUntilVisible` passing it the first item in the list? This way we don't have to use magic numbers, such as 1000.0.

Reply:
Done.